Wire-iOS/Sources/Helpers/SemanticColors.swift

```diff
@@ -123,6 +124,7 @@ public enum SemanticColors {
         static let backgroundRed = UIColor(light: Asset.Colors.red100Light, dark: Asset.Colors.red900Dark)
         static let backgroundPurple = UIColor(light: Asset.Colors.purple100Light, dark: Asset.Colors.purple900Dark)
         static let backgroundTurqoise = UIColor(light: Asset.Colors.turquoise100Light, dark: Asset.Colors.turquoise900Dark)
+        static let separator = UIColor(light: Asset.Colors.gray50, dark: Asset.Colors.gray80)
```

Comment:
May I suggest a more descriptive name for that color? Something like:

```suggestion
        static let backgroundSeparator = UIColor(light: Asset.Colors.gray50, dark: Asset.Colors.gray80)
```

Also the fact that we need a new color for the separator feels a bit weird since we already have a color for that. Let's quickly ask Anette about that and remove that color if necessary.

Reply:
According to Figma, the separator color should be **light: Asset.Colors.gray40, dark: Asset.Colors.gray90**. @netbe Could you double check? We have `View.backgroundSeparatorCell` for this.